kedro/runner/parallel_runner.py

```diff
@@ -52,7 +52,7 @@ def save(self, data: Any):
             # Checks if the error is due to serialisation or not
             try:
                 pickle.dumps(data)
-            except Exception as exc:  # SKIP_IF_NO_SPARK
+            except Exception as exc:  # SKIP_IF_NO_SPARK, pylint: disable=redefined-outer-name
```

Comment:
The latest Pylint update is picking up the inner `exc` variable as an error. To get around this I am making Pylint ignore that error here.